docs/reference/commandline/swarm_leave.md

```diff
@@ -31,12 +34,17 @@ dkp8vy1dq1kxleu9g4u78tlag    worker1   Ready   Active        Reachable
 dvfxp4zseq4s0rih1selh0d20 *  manager1  Ready   Active        Leader
 ```
 
-On a worker node, worker2 in the following example:
+To remove `worker2`, issue the following command from `worker2` itself:
 ```bash
 $ docker swarm leave
 Node left the default swarm.
 ```
 
+To remove the node from the manager, use `docker node rm` instead.
+```bash
+$ docker node rm worker2
+```
```

Comment:
I believe `node rm` can only be used on an active worker if you pass the `--force` argument.